.github/workflows/buildandpush.yml

```diff
@@ -48,9 +48,9 @@ jobs:
                
           echo VERSION=$VERSION
           
-          docker tag $JAVA_IMAGE_NAME $JAVA_IMAGE_ID:${{ github.sha }}
-          docker push $JAVA_IMAGE_ID:${{ github.sha }}
+          docker tag $JAVA_IMAGE_NAME $JAVA_IMAGE_ID
```

Comment:
Can you remove tagging while building docker image?

Reply:
Tagging is required to push the image.

Reply:
I am referring to docker build . --file java/Dockerfile.githubactions --tag $JAVA_IMAGE_NAME --label "runnumber=${GITHUB_RUN_ID}" but was not able to add comment thr